src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -121,33 +136,13 @@
   end
 
   describe 'GET #show' do
-    let(:source_package) { create(:package) }
-    let(:target_package) { create(:package, name: 'target_package', project: factory) }
-    let(:bs_request) do
-      create(:bs_request_with_submit_action,
-             target_project: factory.name,
-             target_package: target_package.name,
-             source_project: source_package.project.name,
-             source_package: source_package.name)
-    end
-    let(:description) do
-      <<-DESCRIPTION
-        requests:
-          - { id: #{bs_request.number} }
-      DESCRIPTION
-    end
-
     context 'with a existent factory_staging_project' do
-      before do
-        bs_request.update(state: 'declined')
-        factory_staging_a.update(description: description)
-      end
-
       context 'requesting html' do
-        subject { get :show, params: { project: factory, project_name: 'A' } }
+        subject! { get :show, params: { project: factory, project_name: 'A' } }
```

Comment:
Instead of `'A'`, let's use `Faker`. Something like `Faker::String.random(10)`

Reply:
staging  projects are capital letters

Reply:
A staging project has a `letter` method

Reply:
As stated above, I don't mind if you change this. But I wonder if a random 10 char string will be more readable than A, B, C, ...

Reply:
@dmarcoux what I hate about Faker is that it normally changes the VCR files, because the endpoint '/projects/Avenging-Disco-Vampires' changed to '/projects/The-Disco-Godfather'. Therefore it appends a cassette to the VCR.. In the staging project, I would keep it as in the workflow covered by the "Staging Dashboard"  https://en.opensuse.org/openSUSE:Factory_development_model